src/modm/driver/inertial/mmc5603.hpp

```diff
@@ -0,0 +1,342 @@
+		Take_meas_M = Bit0,
+		/// Take Measure of Temperature, or TM_T bit. Writing a 1 into this
+		/// location causes the chip to perform a temperature measurement.
+		/// This bit is self-clearing at the end of each measurement.
+		Take_meas_T = Bit1,
+		/// Writing a 1 into this location will cause the chip to do the Set
+		/// operation, which will allow large set current to flow through the
+		/// sensor coils for 375ns. This bit is self-cleared at the end of Set operation.
+		DoSet = Bit3,
+		/// Writing a 1 into this location will cause the chip to do the Reset
+		/// operation, which will allow large reset current to flow through the
+		/// sensor coils for 375ns. This bit is self-cleared at the end of Reset operation.
+		DoReset = Bit4,
+		/// Writing a 1 into this location will enable the function of automatic
+		/// set/reset. This function applies to both on-demand and continuous-time
+		/// measurements. This bit must be set to 1 in order to activate the feature
+		/// of periodic set. This bit is recommended to set to “1” in the application.
+		Auto_SR_en = Bit5,
+		/// Writing a 1 into this location will enable the function of automatic
+		/// self-test. The threshold in register 1EH, 1FH, 20H should be set before
+		/// this bit is set to 1. This bit clears itself after the operation is completed.
+		Auto_st_en = Bit6,
+		/// Writing a 1 into this location will start the calculation of the measurement
+		/// period according to the ODR. This bit should be set before continuous-mode
+		/// measurements are started. This bit is self- cleared after the measurement
+		/// period is calculated by internal circuits.
+		Cmm_freq_en = Bit7,
+	};
+	MODM_FLAGS8(Control0);
+
+	enum class
+	Control1 : uint8_t
+	{
+		Bandwidth_Mask = 0b11,
+		/// Writing “1” will disable this channel, and reduce Measurement Time and
+		/// total charge per measurement.When a channel is disabled it is simply
+		/// skipped during Take Measure routine. Its output register is not reset
+		/// and will maintain the last value written to it when this channel was
+		/// active. Note: Y/Z needs to be inhibited the same time in case needed.
+		X_inhibit = Bit2,
+		Y_inhibit = Bit3,
+		Z_inhibit = Bit4,
+		/// Writing 1 into this location will bring a DC current through the
+		/// self-test coil of the sensor. This current will cause an offset of
+		/// the magnetic field. This function is used to check whether the sensor
+		/// has been saturated.
+		St_enp = Bit5,
+		/// The function of this bit is similar to ST_ENP, but the offset of the
+		/// magnetic field is of opposite polarity.
+		St_enm = Bit6,
+		/// Software Reset. Writing "1" will cause the part to reset, similar to
+		/// power-up. It will clear all registers and also re-read OTP as part
+		/// of its startup routine. The power on time is 20mS.
+		Sw_reset = Bit7,
+	};
+	MODM_FLAGS8(Control1);
+
+	enum class
+	Bandwidth : uint8_t
+	{
+		Ms6_6 = 0x00,
+		Ms3_5 = 0x01,
+		Ms2_0 = 0x02,
+		Ms1_2 = 0x03,
```

Comment:
```suggestion
		Time6600us = 0x00,
		Time3500us = 0x01,
		Time2000us = 0x02,
		Time1200us = 0x03,
```

... could be a little bit more self explaining.

Reply:
Added some docs from the datasheet.